Makefile

```diff
@@ -35,6 +35,14 @@ distclean:
 	$(RM) -r $(BUILDDIR) $(TARGETS)
 	$(ECHO) " done"
 
+# Wrapper targets to enable building man pages / doc.
+# Should do the same as the implicit target at the end.
+man: CMAKE_ARGS+=-D GENERATE_MANPAGES=1
+ldoc: CMAKE_ARGS+=-D GENERATE_DOC=1
+ldoc man: $(BUILDDIR)/Makefile
+	$(ECHO) "Running make $@ in $(BUILDDIR)…"
+	$(MAKE) -C $(BUILDDIR) $@
```

Comment:
`man` and `ldoc` should be added to `.PHONY` at the end of the file.

Also, does this actually work in a non-clean build? CMake was already run, so it will not be rerun and the new `CMAKE_ARGS` are just ignored.

Reply:
You are right.. We could have a `cmake-configure` target for them, but then again this adds some overhead, so I will just remove the helpers: the note from the failing target should be clear enough  then.